code/stable/dblpendulum/.drasil/inverted_seeds.json

```diff
@@ -0,0 +1,2050 @@
+{
```

Comment:
Do we really want to save all this "debug" information in stable?

Reply:
I definitely wouldn't want to stuff every possible artifact in `stable/`, but I think this particular kind might be okay since all Drasil projects will share the same debugging mechanism and because it will allow us to debug changes to the background knowledge as the projects are changed, without looking at the Haskell syntax as often.

That being said, I don't hold a strong opinion about this, and I am definitely open to making it configurable at some level other than Haskell.

Reply:
Let's experiment with it turned on and see how much churn that generates. I am concerned...